lib/html/template/rose-bush/view.html

```diff
@@ -27,19 +27,38 @@
 {% include "job-entry.html" %}
 </div>
 {% endif -%}
+
+<br />
+
+<div class="container-fluid">
+  <div class="form-inline">
+    <div class="input-group stylish-input-group">
+      	<input class="form-control" type="text" name="search" 
+	       id="loggrep" placeholder="Search this file">
+        <label class="radio" style="margin-left:20px">
+	    <input type="radio" name="search_mode" value="TEXT" checked>
+            Text
+	</label>
+	<label class="radio" style="margin-left:10px">
+	    <input type="radio" name="search_mode" value="REGEX">
+	    Regex
+        </label>
```

Comment:
I think we should use a normal HTML _form_ here, and where possible, try to avoid the complexity of having to write JavaScript to event handle _form_ inputs.